openapi/components/schemas/CreditMemos/CommonCreditMemo.yaml

```
@@ -50,8 +50,9 @@ properties:
           List of invoices that the credit memo is allocated to.
 
           - To delete invoice allocations, send an empty invoice allocation array in the request.
-            Paid invoices cannot be modified or deleted.
+            Only `unpaid`, `partially-paid` and `past-due` invoices can be deleted.
           - To modify invoice allocations, send a modified invoice allocation array in the request.
+            Only `unpaid`, `partially-paid` and `past-due` invoices can be modified.
```

nitpick: [serial comma](https://product.rebilly.dev/writing-style-guide/grammar-and-mechanics/commas/#serial-commas)
```suggestion
            Only `unpaid`, `partially-paid`, and `past-due` invoices can be deleted.
          - To modify invoice allocations, send a modified invoice allocation array in the request.
            Only `unpaid`, `partially-paid`, and `past-due` invoices can be modified.
```